src/xcode/ENA/ENA/Source/Scenes/ExposureDetection/ExposureDetectionViewModel.swift

```diff
@@ -41,7 +41,7 @@ class ExposureDetectionViewModel: CountdownTimerDelegate {
 				self?.riskProviderActivityState = $0
 
 				switch $0 {
-				case .downloading:
+				case .downloading, .riskManuallyRequested:
```

Comment:
I'm wondering if need to make the same change in 522 as well and I think we need to extend the `isActive` property of the `RiskProviderActivityState` as well to have the screen correctly configured